docs/guides/LocalLLMs.md

```diff
@@ -54,6 +55,28 @@ docker run \
 
 You should now be able to connect to `http://localhost:3000/`
 
+### 2.2 Build from Source
+Use the instructions in [Development.md](/Development.md) to build OpenDevin.
+Make sure `config.toml` is there by running `make setup-config` which will create one for you. In `config.toml`, enter the followings:
+
+```
+LLM_MODEL="ollama/llama3:8b"
```

Comment:
Instructions above are for `codellama:7b`--we should be consistent (I'm good with that or llama3:8b)

Reply:
tks, addressed in commit [80f15b5](https://github.com/OpenDevin/OpenDevin/pull/1269/commits/80f15b5b38b2cbd91900b92faaf29824a81adb74)